android/java/org/chromium/chrome/browser/toolbar/BraveToolbarManager.java

```diff
@@ -330,6 +331,12 @@ protected void onOrientationChange(int newOrientation) {
             setBottomToolbarVisible(isBottomToolbarVisible);
         }
 
+        if (mActivity instanceof BraveLeoActivity) {
+            // When Leo panel is shown on rotated screen we don't care about
+            // the toolbar.
+            return;
```

Comment:
is there still bottom toolbar when you close Leo Activity after a rotation?

Reply:
No, when I close Leo Activity after a rotation, there is no bottom toolbar. 

But the same behavior is on our `Nightly` and `Stable`, there is no bottom toolbar in landscape orientation  even without Leo.


https://github.com/brave/brave-core/assets/24739341/17b95c0d-bc8f-4493-9eb4-6d0cdbd67333

Reply:
ah yeah, we turn it off on landscape mode, yes, we are good in that case.

Reply:
> I wonder if bottom toolbar is scrolled properly on page scrolling after Leo activity closed as we don't apply adjustment in this case.

@samartnik yes, the bottom toolbar get hidden while scrolling after Leo activity closed, as I understand


https://github.com/brave/brave-core/assets/24739341/4e2f3f9b-1221-48f8-8060-6ecc0e3f162f

Reply:
here how does it work with this plan
1. open several tabs in group to have double toolbar
2. switch to landscape orientation
3. open Leo
4. switch back to portrait orientation and close Leo
5. check weather during scroll the toolbar will be hidden in a normal way


https://github.com/brave/brave-core/assets/24739341/11346199-c61c-4f27-ad0d-a7b9e1a3737d